cubeseed/settings_test.py

```diff
@@ -0,0 +1,20 @@
+from .settings_prod import *
+
+ALLOWED_HOSTS = ["localhost"]
+
+TEST_RUNNER = "redgreenunittest.django.runner.RedGreenDiscoverRunner"
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.sqlite3",
+        "NAME": BASE_DIR / "db.sqlite3",
+    }
+}
+
+REST_FRAMEWORK["TEST_REQUEST_DEFAULT_FORMAT"] = "json"
+REST_FRAMEWORK["TEST_REQUEST_RENDERER_CLASSES"] = [
+    "rest_framework.renderers.JSONRenderer",
```

Comment:
how about handling file uploads of multipart files during testing ; may be can add Multipart renderer
"rest_framework.renderers.MultiPartRenderer" into our framework

Reply:
Just merged from main and it looks like you added that alongside some test, right? 
It is now on test and development settings.